packages/cli/src/template.rs

```diff
@@ -47,6 +49,7 @@ impl Template {
             Template::Svelte => "Svelte - (https://svelte.dev/)",
             Template::React => "React - (https://reactjs.org/)",
             Template::Solid => "Solid - (https://www.solidjs.com/)",
+            Template::Preact => "Preact - (https://preactjs.com/)",
```

Comment:
The order here (especially in regards to Angular) was something I was unsure about: generally I've appended Preact after Solid, as it made sense to group alongside the other `framework` & `framework-ts` style options, but appending it after Solid here means it goes above Angular which probably isn't ideal.

Is it better to keep the consistency (after Solid) or keep order of addition, and move it to the bottom (below Angular)? Happy to do whatever, just let me know!

Reply:
It should be added at the end, new templates always go last

Reply:
Does that go for everything, including places like [this](https://github.com/tauri-apps/create-tauri-app/pull/455/files#diff-f23d9baaac1784128437d225454dc53bb2f2abbe3830ecf5e61c719fda9270e0L69-L74f)?

Reply:
yeah pretty much everywhere

Reply:
Will do then, thanks!

Reply:
Should be good now.